### DISCUSSION TITLE
Welcome to vue-j-scroll.js Discussions!

### DISCUSSION THREAD

**Discussion by DavidJ-0** [DavidJ-0/vue-j-scroll.js#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 2422247006** [comment]
引入npm包报错啊老哥

**Comment by DavidJ-0** [comment]
报错信息发给我&nbsp; 最好有截图和文本 我会及时处理的



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年4月15日(星期四) 下午2:52
收件人: ***@***.***&gt;; 
抄送: "狼 too ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [DavidJ-0/vue-j-scroll.js] Welcome to vue-j-scroll.js Discussions! (#1)





 
引入npm包报错啊老哥
